tests/components/UserActions/UserActions.spec.ts

```diff
@@ -188,6 +188,49 @@ describe('UserActions', () => {
         });
     });
 
+    it('should pass custom init options to each of the sub components', () => {
+        const FAKE_USER_ID = 'someUserId' + Math.random();
+        const initOptions = {} as any;
```

Comment:
here I think you can declare the whole object in a single run. This will provide intellisense and will prevent the use of any 😄 .

Reply:
When declaring the whole object in one shot, I had this compiler error.

    No index signature with a parameter of type 'string' was found on type

Reply:
Yes you need to cast your object in a dict like that `as { [key: string]: any }` to prevent this error this way you said explicitly that you play with a Dict 😃 .

Reply:
🤦‍♂ Thanks

Reply:
Obscure typings for the win 🐈 .